File: PRACTICA PRIMER PARCIAL/Ejercicio23/Python/EliminarDigito.py
x = int(input("Ingrese un número X mayor que 100: "))

if x > 100:
    x_str = str(x)
    min_digit = min(x_str)
    y_str = x_str.replace(min_digit, '', 1)
    y = int(y_str)
    print("Y:", y)
else:
    print("El número X no es mayor que 100.")

File: PRACTICA PRIMER PARCIAL/Ejercicio23/Java/EliminarDigito.java
import java.util.Scanner;
import java.util.Arrays;

public class EliminarDigito {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese un número X mayor que 100: ");
        int x = scanner.nextInt();

        if (x > 100) {
            String xStr = Integer.toString(x);
            char[] digits = xStr.toCharArray();
            Arrays.sort(digits);
            char minDigit = digits[0];
            String yStr = xStr.replaceFirst(Character.toString(minDigit), "");
            int y = Integer.parseInt(yStr);
            System.out.println("Y: " + y);
        } else {
            System.out.println("El número X no es mayor que 100.");
        }
    }
}